exist-core/src/main/java/org/exist/repo/Deployment.java

```diff
@@ -392,6 +393,13 @@ private void checkProcessorVersion(final PackageLoader.Version version) throws P
                     targetCollection = XmldbURI.SYSTEM.append("repo/" + pkgColl);
                 }
 
+                // skip trigger execution and remember previous state
+                final Optional<ElementImpl> skipTriggers = findElement(repoXML, SKIP_TRIGGERS_ELEMENT);
+                final boolean triggerState = broker.isTriggersEnabled();
```

Comment:
more compact would be `skipTriggers.ifPresent(c -> broker.setTriggersEnabled(false));`

Reply:
Thanks, will consider. For now, too much compaction kills debugging